Checklist

```diff
@@ -0,0 +1,24 @@
+<!-- Check all that apply [x] -->
+
+# Checklist
+
+This is a checklist that can be used in effort to review before submitting a PR, in the Edge Drivers Repo.
+
+- [ ] This is the first time I am contributing code:
```

Comment:
Is this entire checklist present on every PR? If so, I think we should remove the entire first bullet and sub bullets about being the first contributions, as it wont be used very often and we can tell most of this through the github bot that checks for the CLA.

Reply:
Yes - I will be used for every PR. 

The goal of adding those is trying to get the partner to read through those before submitting, with the goal to decrease feedback loops.